artemis/http_requests.py

```diff
@@ -84,7 +85,7 @@ def _internal_request() -> HTTPResponse:
             verify=False,
             stream=True,
             timeout=Config.Limits.REQUEST_TIMEOUT_SECONDS,
-            headers=HEADERS,
+            headers=HEADERS if kwargs.get("headers") is None else kwargs.get("headers"),
```

Comment:
imo:

```
headers = copy.copy(HEADERS)
if 'headers' in kwargs:
    headers.update(kwargs['headers'])
```

that way if someone sets headers={"X-Whatever": "lalalala"}, they'll get updated with base ones